### DISCUSSION TITLE
Show 100s and 50s in the timeline

### DISCUSSION THREAD

**Discussion by developomp** [ppy/osu#18096] [Ideas] [answered]
It'd be nice to have the location of 100s and 50s dropped in the timeline similar to what cpol does.

Example:
![image](https://user-images.githubusercontent.com/40858122/166706788-76543359-38a3-4093-8e71-62d51f046736.png)

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/5681
